drf_model_pusher/providers.py

```diff
@@ -74,12 +74,7 @@ def _sync_cache(self):
         occupied_channels = response.get("channels", {}).keys()
 
         cache.set_many(
-            list(
-                map(
-                    lambda occupied_channel: {"drf-model-pusher:occupied:{}".format(occupied_channel): True},
-                    occupied_channels
-                )
-            )
+            dict.fromkeys(occupied_channels, True)
```

Comment:
You still need to update the keys to be prefixed with `drf-model-pusher:occupied:`

Reply:
Have updated